### ISSUE TITLE
Why does validation set accuracy fluctuate so much?

### ISSUE THREAD

**Issue by QAQEthan** [brain-bzh/dcase-2020-task1-subtaskB#1]
The accuracy of training set increases slowly, but the accuracy of validation set fluctuates greatly
